README.rst

```diff
@@ -1,43 +1,112 @@
-===============================
-DateParser
-===============================
+====================================================
+dateparser -- python parser for human readable dates
+====================================================
 
-.. image:: https://badge.fury.io/py/dateparser.png
-    :target: http://badge.fury.io/py/dateparser
+.. image:: https://img.shields.io/travis/scrapinghub/dateparser/master.svg?style=flat-square
+    :target: https://travis-ci.org/scrapinghub/dateparser
+    :alt: travis build status
 
-.. image:: https://travis-ci.org/scrapinghub/dateparser.png?branch=master
-        :target: https://travis-ci.org/scrapinghub/dateparser
+.. image:: https://img.shields.io/pypi/dd/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser/
+    :alt: pypi downloads per day
 
-.. image:: https://pypip.in/d/dateparser/badge.png
-        :target: https://pypi.python.org/pypi/dateparser
+.. image:: https://img.shields.io/pypi/v/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser
+    :alt: pypi version
 
 
-Date parsing library designed to make it easy parsing dates commonly found in web pages
+`dateparser` provides modules to easily parse localized dates in almost
+any string formats commonly found on web pages.
 
 
-* Free software: BSD license
-* Documentation: https://dateparser.readthedocs.org.
-
 Features
 --------
+* Generic parsing of dates in English, Spanish, Dutch, Russian and several other langauges and formats.
+* Generic parsing of contextual dates like: ``'1 min ago'``, ``'2 weeks ago'``, ``'3 months, 1 weeks and 1 day ago'``.
+* Generic parsing of dates with time zones abbreviations like: ``'August 14, 2015 EST'``, ``'July 4, 2013 PST'``.
+* Easy confingurations to change default parsing behavior. See :mod:`dateparser.conf`.
```

Comment:
Please move any configuration related documentation to separate branch. Configuration in fetus condition here and we do not want to encourage people to use it as it is because interface will change in the future. We are landing configuration in 0.3 release.